docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -1,216 +1,339 @@
+-   An operating system selection (for example, a Linux distribution, Windows Nano Server, or Windows Server Core).
 
-- Files added by the developer (application binaries, etc.).
+-   Files added by the developer (application binaries, etc.).
 
-- Configuration information (environment settings and dependencies).
+-   Configuration information (environment settings and dependencies).
 
 ## Workflow for developing Docker container-based applications
 
-This section describes the *inner-loop* development workflow for Docker container-based applications. The inner-loop workflow means it is not taking into account the broader DevOps workflow and just focuses on the development work done on the developer’s computer. The initial steps to set up the environment are not included, since those are done only once.
+This section describes the *inner-loop* development workflow for Docker container-based applications. The inner-loop workflow means it is not considering the broader DevOps workflow (that can include up to production deployment) and just focuses on the development work done on the developer's computer. The initial steps to set up the environment are not included, since those are done only once.
 
 An application is composed of your own services plus additional libraries (dependencies). The following are the basic steps you usually take when building a Docker application, as illustrated in Figure 5-1.
 
-![Step-by-step workflow for developing Docker containerized apps graphic](./media/image1.png)
+![The development process for Docker apps: 1 - Code your App, 2 - Write Dockerfile/s, 3 - Create images defined at Dockerfile/s, 4 - (optional) Compose services in the docker-compose.yml file, 5 - Run container or docker-compose app, 6 - Test your app or microservices, 7 - Push to repo and repeat. ](./media/image1.png)
 
 **Figure 5-1.** Step-by-step workflow for developing Docker containerized apps
 
-In this guide, this whole process is detailed and every major step is explained by focusing on a Visual Studio environment.
+In this section, this whole process is detailed and every major step is explained by focusing on a Visual Studio environment.
 
-When you are using an editor/CLI development approach (for example, Visual Studio Code plus Docker CLI on macOS or Windows), you need to know every step, generally in more detail than if you are using Visual Studio. For more information about working in a CLI environment, see the e-book [Containerized Docker Application lifecycle with Microsoft Platforms and Tools](http://aka.ms/dockerlifecycleebook/).
+When you're using an editor/CLI development approach (for example, Visual Studio Code plus Docker CLI on macOS or Windows), you need to know every step, generally in more detail than if you are using Visual Studio. For more details about working in a CLI environment, refer to the e-book [Containerized Docker Application lifecycle with Microsoft Platforms and Tools](https://aka.ms/dockerlifecycleebook/).
```

Comment:
The original sentence "For more information..., see" should be kept. It's part of our style guide on how we refer to content
you are -> you're

Reply:
done.